app/views/search/index.html.erb

```diff
@@ -110,6 +110,8 @@
           <p class="collection-filters">
             <input type="checkbox" id='has-granules' data-bind="checked: query.hasGranules"/>
             <label for="has-granules">Include collections without granules</label>
```

Comment:
@mark inverting the logic was easy up until making the URL parameters work.  The inverted has_granules checkbox works - it's the page refresh that's not working well.  Since it is inverted that means it needs to take an action when its state is *false* - not true - and somehow this isn't syncing right with how URL parameters are handled.

The exact same issue is also causing the new non-EOSDIS toggle to not handle a page refresh well, either.  As @mightynimble requested - I added code so that it is seen in the URL.  However, like the inverted hasGranules, it adds something to the URL only when it is *unchecked*.  When it is checked, nothing is added to the URL - it is only when unchecked that an action is taken.  This little inverted spaghetti knot is causing both checkboxes to get confused on page refreshes.

I'll work on it a bit more (though have an errand to run shortly) and will get back to you two later today.